src/types/global.d.ts

```diff
@@ -0,0 +1,15 @@
+import BitmovinPlayer from 'bitmovin-player';
+import ConvivaAnalytics from '../ts/ConvivaAnalytics';
+
+declare global {
+  interface Window {
+    bitmovin: {
+      player?: {
+        Player?: typeof BitmovinPlayer,
+        analytics?: {
+          ConvivaAnalytics?: typeof ConvivaAnalytics,
```

Comment:
Global exports are undesired and only a fallback if the JS file is loaded directly into the webpage without a module loader, in which case it will be JS (not TS), so type declarations for the global export are unnecessary and discouraged.